index.php

```diff
@@ -115,7 +126,7 @@
                     <div class="lot__state">
                         <div class="lot__rate">
                             <span class="lot__amount">Стартовая цена</span>
-                            <span class="lot__cost"><?=$lot['price']; ?>₽</span>
+                            <span class="lot__cost"><?php print(format_price($lot['price'])); ?></span>
```

Comment:
По-моему, в требованиях этого нет, но я крайне рекомендую использовать `echo` вместо `print` для вывода текста. Хороший пример неожиданного поведения print: https://www.php.net/manual/ru/function.print.php#85310

А ещё, для `echo` есть очень удобное сокращение в шаблонах: `<?= $var; ?>`. Я сейчас добавлю suggestion с изменением с таким синтаксисом:
```suggestion
                            <span class="lot__cost"><?= format_price($lot['price']); ?></span>
```